javascriptv3/README.md

```diff
@@ -37,13 +37,26 @@ Cross-service examples are located in the [_cross-services folder_](./cross-serv
 
 **Note**: Running the tests might result in charges to your AWS account.
 
-1. Where tests are provided, navigate to the 'tests' folder located in the root of the service folder.
+Tests can be run for every service in this repository, or for a specific service. Tests are
+either unit tests or integration tests. You may choose to run only unit tests, only integration
+tests, or both.
```

Comment:
You can run tests for a specific service, or for every service in this repository. Choose whether to run unit tests, integration tests, or both.